### ISSUE TITLE
widget seems to require to load the breakable library

### ISSUE THREAD

**Issue by FrankMittelbach** [T-F-S/tcolorbox#117]
Running
```
\documentclass{article}
\pagestyle{empty}
\setlength\textwidth{159.0pt}

\usepackage{tcolorbox}
\tcbuselibrary{skins}

\begin{document}
\begin{tcolorbox}[widget,title=My title]
  This is my content. \tcblower Second part.
\end{tcolorbox}
\end{document}
```
gives an error about 
```
Package pgfkeys Error: I do not know the key '/tcb/toprule at 
break', to which you passed '0mm', and I am going to ignore it. Perhaps you mis
spelled it.
```
which can be easily repaired by loading the `breakable` library but it look unintentional to me (and not documented).

**Comment by muzimuzhi** [comment]
Style key `widget` wrongly used `toprule at break` key, which seems can be safely removed.
https://github.com/T-F-S/tcolorbox/blob/78a915c1b1971a82cd3cce09f6008ccc065ab625/tex/latex/tcolorbox/tcbskins.code.tex#L774

**Comment by T-F-S** [comment]
I will remove `toprule at break`, because it is unintentional. I guess it is a remnant from a test.
Unfortunately, the removal may change the geometry of broken `widget` boxes with possible side effects for page breaking.
Nevertheless, I will remove it.

**Comment by T-F-S** [comment]
Fixed for version 4.40 (2020/09/25).
